packages-exp/firebase-exp/rollup.config.js

```diff
@@ -129,6 +129,38 @@ const componentBuilds = pkg.components
     // It is needed for handling sub modules, for example firestore/lite which should produce firebase-firestore-lite.js
     // Otherwise, we will create a directory with '/' in the name.
     const componentName = component.replace('/', '-');
+
+    if (component === 'messaging') {
+      return [
+        {
+          input: `${component}/index.cdn.ts`,
```

Comment:
Do you mean `index.ts` here? We only need to build cjs and es targets for the client SDK.

Reply:
Applies to the release build config too.